open-metadata-resources/open-metadata-labs/egeria-server-config.ipynb

```diff
@@ -892,9 +892,89 @@
     "\n",
     "print (\"Configuring the Integration View Services\")\n",
     "\n",
-    "# For now rex (repository explorer) follows the same pattern as glossary author, but this will change to be more similar to dino & tex\n",
+    "# repository explorer integration view service\n",
     "viewService=\"rex\"\n",
-    "configureGovernanceSolutionViewService(adminPlatformURL, adminUserId, viewServerName, viewService, remotePlatformURL,remoteServerName)\n",
+    "rexConfigBody = {\n",
+    "            \"class\":\"IntegrationViewServiceConfig\",\n",
+    "            \"viewServiceAdminClass\":\"org.odpi.openmetadata.viewservices.rex.admin.RexViewAdmin\",\n",
+    "            \"viewServiceFullName\":\"Type Explorer\",\n",
+    "            \"viewServiceOperationalStatus\":\"ENABLED\",\n",
+    "            \"omagserverPlatformRootURL\": \"UNUSED\",\n",
+    "            \"omagserverName\" : \"UNUSED\",\n",
+    "            \"resourceEndpoints\" : [\n",
+    "                {\n",
+    "                    \"class\"              : \"ResourceEndpointConfig\",\n",
+    "                    \"resourceCategory\"   : \"Platform\",\n",
+    "                    \"description\"        : \"Core Platform\",\n",
+    "                    \"platformName\"       : \"Core\",\n",
+    "                    \"platformRootURL\"    : corePlatformURL\n",
+    "                },\n",
+    "                {\n",
+    "                    \"class\"              : \"ResourceEndpointConfig\",\n",
+    "                    \"resourceCategory\"   : \"Platform\",\n",
+    "                    \"description\"        : \"DataLake Platform\",\n",
+    "                    \"platformName\"       : \"DataLake\",\n",
+    "                    \"platformRootURL\"    : dataLakePlatformURL\n",
+    "                },\n",
+    "                {\n",
+    "                    \"class\"              : \"ResourceEndpointConfig\",\n",
+    "                    \"resourceCategory\"   : \"Platform\",\n",
+    "                    \"description\"        : \"Development Platform\",\n",
+    "                    \"platformName\"       : \"Development\",\n",
+    "                    \"platformRootURL\"    : devPlatformURL\n",
+    "                },\n",
+    "                {\n",
+    "                    \"class\"              : \"ResourceEndpointConfig\",\n",
+    "                    \"resourceCategory\"   : \"Server\",\n",
+    "                    \"serverInstanceName\" : \"cocoMDS1\",\n",
```

Comment:
Whilst this configuration is valid - I wonder whether it might be more illustrative if we distinguish the serviceInstanceName from the serviceName. If there is only ever one instance of the service then it genuinely makes no difference and is arguably clearer the way it is here (with the two fields having the same value). If there *might* be more than one instance then reflecting that in the serviceInstanceName is clearer.